war/src/main/js/widgets/config/tabbar.less

```diff
@@ -50,7 +50,7 @@
 
   .form-config.tabBarFrame {
     position: relative;
-    border-bottom: solid 1px var(--input-border);
+    border-bottom: solid 1px var(--medium-grey);
```

Comment:
not sure if I agree on this being a medium-grey instead of another variable, it breaks dark theme quite badly, and dark-theme already redefines this colour.

can we use a border variable instead to give more control, if input border isn't right then another one is fine

Reply:
I can try and play with other colours. FWIW medium-grey was always intended to be for borders, as it does not provide enough contrast with text.

For a more long-term solution, is there no chance that the dark theme can disambiguate the greys?

Reply:
For context, the dark theme definition of the grayscale is as follows:

```
  --background: var(--dark-theme-bg-dark);
  --very-light-grey: var(--dark-theme-bg-dark);
  --light-grey: var(--dark-theme-bg-333);
  --medium-grey: var(--dark-theme-bg-333);
  --white: var(--dark-theme-bg-333);
```

These are just 2 colours to replace 5. If we need to always avoid breaking dark theme it will be quite difficult to take advantage of the grayscale. Also, using another colour kind of defeats the point of having a palette to begin with IMO.

Reply:
we can try fiddle in dark theme a bit and see if we can get something sensible, but the other option that I suggested was just to define a variable that allows more finer control:


something like:
`--container-border: var(--medium-grey)`

Reply:
Would you suggest making this one a stable variable in the official-ish theme? In what other places would you suggest using it?